client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -158,16 +167,25 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 	}
 
 	const langFragment = lang ? `/${ lang }` : '';
+	const anchorFmParameter = isAnchorFmSignup ? `&anchor_podcast=${ anchorFmPodcastId }` : '';
 	const loginRedirectUrl = encodeURIComponent(
-		`${ window.location.origin }/new${ makePath( Step.CreateSite ) }?new`
+		`${ window.location.origin }/new${
+			isAnchorFmSignup ? makePath( Step.IntentGathering ) : makePath( Step.CreateSite )
+		}?new${ anchorFmParameter }`
 	);
 	const signupUrl = encodeURIComponent( `/new${ makePath( Step[ currentStep ] ) }?signup` );
 	const loginUrl = `/log-in/new${ langFragment }?redirect_to=${ loginRedirectUrl }&signup_url=${ signupUrl }`;
+	const modalTitle = isAnchorFmSignup
+		? __( 'Create your podcast site with WordPress.com' )
+		: __( 'Save your progress' );
+	const modalSubTitle = isAnchorFmSignup
+		? __( 'Create a WordPress.com account and start creating your free site.' )
+		: __( 'Enter an email and password to save your progress and continue.' );
 
 	return (
 		<Modal
 			className={ 'signup-form' }
-			title={ __( 'Save your progress' ) }
+			title={ modalTitle }
```

Comment:
I thought I read someone that these couldn't be translated dynamically? If I'm mistaken, please ignore

Reply:
I moved the translations into the ternary based on https://github.com/Automattic/wp-calypso/pull/47600#discussion_r533521894

The `modalTitle` is used twice so a variable is convenient, but maybe it would be better to just eliminate them altogether, [based on these examples](https://github.com/Automattic/i18n-calypso/blob/HEAD/README.md).

Reply:
I [removed the variables](https://github.com/Automattic/wp-calypso/pull/47600/commits/56edf58cc8298035da7ed43f8d8db2c0904b156b).

Reply:
100% agree with you to keep this dry -- I just wasn't sure if someone mentioned that having a translated variable would work as expected